### PR TITLE
Prometheus metrics

### PR DESCRIPTION
**Pull request by JumasJM**
#### Checklist

- [ ] Title of the PR starts with chart name (e.g. `[krakend]`)
- [ ] Update the documentation if a new value has been added
- [ ] Update chart release notes *[annotation](https://artifacthub.io/docs/topics/annotations/helm/)*

### CODE REVIEW

File: charts/frinx-machine-monitoring/config/loki_logging.json [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "current": {
+          "selected": false,
+          "text": "workflow-manager-schellar",
+          "value": "workflow-manager-schellar"
+        },
```
Comment (Jozefiel) [inline charts/frinx-machine-monitoring/config/loki_logging.json:276]: this should be removed

File: charts/frinx-machine-monitoring/config/loki_uniconfig_device_logging.json [hunk 2]
```diff
@@ -151,7 +155,7 @@
             "uid": "P8E80F9AEF21F6940"
           },
           "editorMode": "code",
-          "expr": "{app=\"$app\", container=\"$container\"} |~ \"$search\"",
+          "expr": "{app=\"uniconfig\", container=\"uniconfig\", deviceName=\"$deviceName\"} |~ \"$search\"",
```
Comment (Jozefiel) [inline charts/frinx-machine-monitoring/config/loki_uniconfig_device_logging.json:158]: this is locked to zone uniconfig, but we can deploy another UC zone with another name

File: charts/inventory/README.md [hunk 3]
```diff
@@ -78,4 +76,6 @@ helm uninstall [RELEASE_NAME]
 | serviceAccount.create | bool | `true` | Specifies whether a service account should be created |
 | serviceAccount.name | string | `""` | The name of the service account to use. If not set and create is true, a name is generated using the fullname template |
 | tolerations | list | `[]` | [Tolerations for pod assignment](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) |
-
+| utilitiesImage.pullPolicy | string | `"IfNotPresent"` | Image pull policy |
+| utilitiesImage.repository | string | `"frinx/utilities-alpine"` | utilities image repository |
```
Comment (Jozefiel) [inline charts/inventory/README.md:80]: Make utilities with first letter capital

File: charts/workflow-manager/README.md [hunk 4]
```diff
@@ -39,24 +39,26 @@ helm uninstall [RELEASE_NAME]
...
...KFLOWOUTPUTPAYLOADSIZETHRESHOLD":85,"CONDUCTOR_ARCHIVE_DB_ENABLED":true,"CONDUCTOR_ARCHIVE_DB_INDEXER_POLLINGINTERVAL":10,"CONDUCTOR_ARCHIVE_DB_INDEXER_THREADCOUNT":4,"CONDUCTOR_ARCHIVE_DB_TYPE":"postgres","CONDUCTOR_DB_TYPE":"postgres","CONDUCTOR_EXTERNALPAYLOADSTORAGE_POSTGRES_CONDUCTORURL":"http://conductor-server:8080","CONDUCTOR_EXTERNALPAYLOADSTORAGE_POSTGRES_MAXDATADAYS":0,"CONDUCTOR_EXTERNALPAYLOADSTORAGE_POSTGRES_MAXDATAMONTHS":0,"CONDUCTOR_EXTERNALPAYLOADSTORAGE_POSTGRES_MAXDATAROWS":100000,"CONDUCTOR_EXTERNALPAYLOADSTORAGE_POSTGRES_MAXDATAYEARS":1,"CONDUCTOR_EXTERNALPAYLOADSTORAGE_TYPE":"postgres","CONDUCTOR_INDEXING_ENABLED":false,"CONDUCTOR_METRICSDATADOG_ENABLED":false,"CONDUCTOR_METRICSLOGGER_ENABLED":true,"CONDUCTOR_METRICSLOGGER_REPORTPERIODSECONDS":15,"CONDUCTOR_METRICSPROMETHEUS_ENABLED":true,"CONDUCTOR_WORKFLOWEXECUTIONLOCK_TYPE":"local_only","CONDUCTOR_WORKFLOWREPAIRSERVICE_ENABLED":false,"LOADSAMPLE":false,"MANAGEMENT_ENDPOINTS_WEB_EXPOSURE_INCLUDE":"prometheus","SPRING_AUTOCONFIGURE_EXCLUDE":"org.springframework.boot.autoconfigure.security.servlet.UserDetailsServiceAutoConfiguration","SPRING_SEARCHDATASOURCE_HIKARI_MAXIMUMPOOLSIZE":8}` | Conductor extra ENV |
```
Comment (Jozefiel) [inline charts/workflow-manager/README.md:45]: Can you improve formating? 
Maybe make annotations or ignore the whole config

File: charts/frinx-machine-monitoring/config/loki_logging.json (hunk 1 above)
Comment (JumasJM) [reply charts/frinx-machine-monitoring/config/loki_logging.json]: removed

File: charts/inventory/README.md (hunk 3 above)
Comment (JumasJM) [reply charts/inventory/README.md]: fixed

File: charts/workflow-manager/README.md (hunk 4 above)
Comment (JumasJM) [reply charts/workflow-manager/README.md]: ignored

**Review by Jozefiel — APPROVED** [review]

**Pull request merged**